### PR TITLE
Add private keys

### PR DESCRIPTION
**Pull request by Bill-Keuntje**
testing keys needed for python and javascript UI
Closxes issue #93

### CODE REVIEW

File: ReadcapURI.json [hunk 1]
```diff
@@ -1 +1 @@
-{"ReadcapURI" : "rho:id:ygi8f15fdt83uwyi7dq4k8ij7z7w4sb9qthq9jcqq76wnpzok9ausx"}
+{"ReadcapURI" : "rho:id:45qoa694ijuiy6enn5dne3eca8ipechh7si3fjzammp1ycz1tcijc6"}
```
Comment (dckc) [inline ReadcapURI.json:1]: oops... `ReadcapURI` got committed.

**Review by dckc — CHANGES_REQUESTED** [review]
oops. `ReadcapURI` got committed.

File: ReadcapURI.json (hunk 1 above)
Comment (Bill-Keuntje) [reply ReadcapURI.json:1]: > oops... `ReadcapURI` got committed.

Yep. How do I remove it?

Comment (dckc) [reply ReadcapURI.json:1]: git commit --amend

Or the equvalent

Comment (Bill-Keuntje) [reply ReadcapURI.json]: updated by adding two new commits reverting the changes

**Comment by Bill-Keuntje** [comment]
@w2vy Please let me know if these changes are suitable to your pyrchain/pytest testing.

**Comment by Bill-Keuntje** [comment]
@dckc New commit reverts incorrectly committed files.

**Comment by w2vy** [comment]
Repalced by https://github.com/rchain-community/rgov/pull/177

**Pull request closed without merging**